src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => (
+
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+
+    {person.isMarried ? (
+      <p className="Person__partner">
+        {person.sex === 'm' ? (
```

Comment:
don't use nested ternary, move this part to separate variable and store needed string there